processor/src/trace/tests/chiplets/hasher.rs

```diff
@@ -29,6 +30,9 @@ const DECODER_HASHER_STATE_RANGE: Range<usize> = range(
     vm_core::decoder::NUM_HASHER_COLUMNS,
 );
 
+const DECODER_OP_BITS_RANGE: Range<usize> =
+    range(DECODER_TRACE_OFFSET + OP_BITS_OFFSET, NUM_OP_BITS);
+
```

Comment:
note: we'll probably want this in core later, since we'll need it for constraints. I think it's fine to have it here now, since we're only using it for tests, but we don't want to forget that it's here and end up having it defined twice. Maybe we should add a TODO? Whatever you think though, I don't have a strong opinion

Reply:
I decided to move this into core.  I think it's better we do this now to avoid missing it in the future as you mentioned.